### ISSUE TITLE
요구사항 정의 확정

### ISSUE THREAD

**Issue by h2dj** [team-durumi/kfem#12]
https://docs.google.com/spreadsheets/d/1hrd-3Www7UBdrmPx2_XJv2LrYtiLz1ha3I67yElwQXM/edit?usp=sharing
환경운동연합에서 1차로 작성해주신 문서입니다.
우리측에서 검토해서 가능/불가능/대안 등 제시해주면 되겠습니다

이후에 여기 이슈 리스트에 정리된 것을 각각 올려두면 되는 것이겠죠?

**Comment by woonjjang** [comment]
지역에서만 요청하셨던 사항에 대해서 중앙에도 남겨놓으신 부분이 있어보이네요. 
1차로 제가 체크해 보았습니다. @mozodev @suncrud 의견들 주세요. 

@h2dj 저 엑셀은 고객 요청사항이고, 나머지 이슈들은 그냥 깃허브에 적어놓으면 되지 않을까요 ? 다시 엑셀에 적는게...

**Comment by h2dj** [comment]
네 제가 "이후에 여기 이슈 리스트에 정리된 것을 각각 올려두면 되는 것이겠죠?" 말씀드린 것이 정리된 이슈는 깃허브에 적어도 되겠다는 것이고요, 고객 요청사항에 대해 응답하기 전에 논의가 필요한 게 있으면 현재 최예지님이 깃헙을 안 쓰시므로 카톡이나 회의 혹은 엑셀 문서를 임시로 써서 소통할 수 있겠지요
